### PR TITLE
9751 cis page build

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Builds /security/cis page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/cis
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1bSv8lV9BJoBYh5yog2eYKtvNMitKZetSSBQ3k2Cu5Cw/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9751

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @petesfrench, can we do something to avoid the icon widow? Perhaps a copy change @SirSamTumless?

image

Also, can we keep the tick icons consistent with the [other security page](https://github.com/canonical-web-and-design/ubuntu.com/pull/9821) please.

File: templates/security/cis.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="u-fixed-width">
+      <h2>Which versions of Ubuntu have tooling for CIS benchmarks?</h2>
+      <p>Canonical provides OpenSCAP content for auditing systems for CIS benchmark compliance, as well as tooling to automate compliance.</p>
+    </div>
+    <table>
```
Comment (bethcollins92) [inline templates/security/cis.html:62]: Make sure this table is inside a `u-fixed-width` or a `row` and `col`

File: templates/security/cis.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    <div class="col-6">
+      <h3>Configure and apply CIS hardening rules in minutes</h3>
+      <p>CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides CIS automation tooling available to its Ubuntu Advantage customers and Ubuntu Pro users on the public cloud.</p>
+      <p>The compliance tooling has two objectives: it lets our customers harden their Ubuntu systems effortlessly and then quickly audit those systems against the published CIS Ubuntu benchmarks. The SCAP content for audit tooling that scans the system for compliance is CIS certified.</p>
+      <a href="https://www.youtube.com/watch?v=wyEX0eyoK88" class="p-button--neutral p-link--external">Watch the video</a>
+    </div>
+  </div>
+</section>
+
+
```
Comment (bethcollins92) [inline templates/security/cis.html:143]: Here you can copy the marketo info from the bottom of the [/security/fips](https://github.com/canonical-web-and-design/ubuntu.com/pull/9852/files) page

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Nice! 👍🏼  - just some small changes. (And make sure to remove the changes from the livepatch page)

File: templates/openstack/cis.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      <p>The CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides tooling for audit and automated compliance with the CIS benchmarks with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
+      <a href="/support#get-in-touch" class="p-button--positive">Access the tooling for audit and automated CIS compliance</a>
```
Comment (bethcollins92) [inline templates/openstack/cis.html:17]:
```suggestion
      <a href="/support/get-in-touch" class="p-button--positive">Access the tooling for audit and automated CIS compliance</a>
```

Comment (bethcollins92) [inline templates/openstack/cis.html:17]:
```suggestion
      <a href="/support/get-in-touch" class="p-button--positive js-invoke-modal">Access the tooling for audit and automated CIS compliance</a>
```

File: templates/openstack/cis.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    <div class="col-6">
+      <h3>Configure and apply CIS hardening rules in minutes</h3>
+      <p>CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides CIS automation tooling available to its Ubuntu Advantage customers and Ubuntu Pro users on the public cloud.</p>
+      <p>The compliance tooling has two objectives: it lets our customers harden their Ubuntu systems effortlessly and then quickly audit those systems against the published CIS Ubuntu benchmarks. The SCAP content for audit tooling that scans the system for compliance is CIS certified.</p>
+      <a href="https://www.youtube.com/watch?v=wyEX0eyoK88" class="p-button--neutral p-link--external">Watch the video</a>
+    </div>
+  </div>
+</section>
+
+
```
Comment (bethcollins92) [inline templates/openstack/cis.html:143]: copy the marketo info from the bottom of this [page](https://github.com/canonical-web-and-design/ubuntu.com/pull/9852/files)

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Looking good! Can you check the rows and cols again and let me know then I'll have another look!

File: templates/security/cis.html [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+      <p>The CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides tooling for audit and automated compliance with the CIS benchmarks with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
+      <a href="/support/get-in-touch" class="p-button--positive">Access the tooling for audit and automated CIS compliance</a>
```
Comment (bethcollins92) [inline templates/security/cis.html:17]: Can you add this `js-invoke-modal` class to the button, it should be a pop up modal when clicked. 
```suggestion
      <a href="/support/get-in-touch" class="p-button--positive js-invoke-modal">Access the tooling for audit and automated CIS compliance</a>
```

**Review by bethcollins92 — APPROVED** [review]
LGTM 👍🏼  Can you just add the modal class to the button and let me know when you've done it, I'll check that before I give the QA +1

**Comment by anasereijo** [comment]
@petesfrench sorry last minute change:
can you remove the italics from 'Comply with the most widely accepted baseline' and make it a <h4>? Thanks

**Comment by bethcollins92** [comment]
Nice! LGTM

File: templates/security/cis.html [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+      <p>The CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides tooling for audit and automated compliance with the CIS benchmarks with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
+      <a href="/support/get-in-touch" class="p-button--positive js-invoke-modal">Access the tooling for audit and automated CIS compliance</a>
```
Comment (anthonydillon) [inline templates/security/cis.html:17]: Is this an actual page? The href acts as a no-js fall back. It should probably go to "/support/contact-us"

File: templates/security/cis.html [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+      <h3>Configure and apply CIS hardening rules in minutes</h3>
+      <p>CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides CIS automation tooling available to its Ubuntu Advantage customers and Ubuntu Pro users on the public cloud.</p>
+      <p>The compliance tooling has two objectives: it lets our customers harden their Ubuntu systems effortlessly and then quickly audit those systems against the published CIS Ubuntu benchmarks. The SCAP content for audit tooling that scans the system for compliance is CIS certified.</p>
+      <a href="https://www.youtube.com/watch?v=wyEX0eyoK88" class="p-button--neutral p-link--external">Watch the video</a>
+    </div>
+  </div>
+</section>
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/support" data-form-id="1257" data-lp-id="2154" data-return-url="https://www.ubuntu.com/contact-us/form/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
```
Comment (anthonydillon) [inline templates/security/cis.html:118]: I think the return URL here should be to support/thank-you

**Pull request merged**